### DISCUSSION TITLE
Installation problem

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#310] [Bug]
Hi,

I have installed DAFoam 3 from source ubuntu and exactly followed the commands in installation tutorial, but in running test I have faced a problem. I would appreciate if you help me with this error.
![Untitled](https://user-images.githubusercontent.com/97816713/170284860-8e392eed-c9a4-4c4a-b286-8f6654660d42.png)

**Comment by friedenhe** [comment]
Check if this helps: https://dafoam.github.io/mydoc_get_started_faq.html#how-to-fix-the-no-module-named-dafoampydasolver-error

**Reply by hosseinnegahban** [reply]
I am compiling the same version of python that is in the tutorial, I have not changed anything. I am still wondering where is the issue.

**Reply by hosseinnegahban** [reply]
I can not find the file pyDASolverIncompressible.cpython-38-x86_64-linux-gnu.so or compressible one

**Reply by hosseinnegahban** [reply]
I dont know why, but after recompiling DAFoam these files showed up at first compile they did not

**Reply by sachinsreedhar** [reply]
I have a similar issue after compilation
This is my bashrc file
![image](https://github.com/mdolab/dafoam/assets/138503664/0bf0ff86-c460-4477-aced-1e0b84e61e85)
Initially i get when i type  DASimpleFoamReverseAD -help
![image](https://github.com/mdolab/dafoam/assets/138503664/3cc4f2f3-066e-45b4-a41e-9d6f152d4078)
When i type blockMesh -help
it says command not found

Then I sourced the loadDafoam.sh
Now my DASimpleFoamReverseAD -help
says no command found 
But my blockMesh works

**Comment by friedenhe** [comment]
Use these commands to switch back to the original version of DAFoam (they are at the end of the OpenFOAM compilation section: https://dafoam.github.io/mydoc_installation_source.html#openfoam ):

<pre>
unset WM_CODI_AD_MODE && \
. $HOME/dafoam/loadDAFoam.sh
</pre>

The unset call is important. Otherwise, you will be still in the ADR mode and it will cause a lot of issues.
